recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "nnpops" %}
+{% set version = "v0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikemhenry/NNPOps/archive/refs/tags/v0.01-pre.1.tar.gz
+  #url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 87f1af4dc34b00d69fd1c0faee2cf5f23f52b48b93cb8f275415037f2a220573
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64 or ppc64le]
+  #string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
```

Comment:
```suggestion
    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
    - sysroot_linux-64  2.17  # [linux64]
```
Try this?

Reply:
I really think you need this:
```
/home/conda/staged-recipes/build_artifacts/nnpops_1642449018632/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/9.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/conda/staged-recipes/build_artifacts/nnpops_1642449018632/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.9/site-packages/torch/lib/libtorch_cpu.so: undefined reference to `memcpy@GLIBC_2.14'
```

pytorch was built with the newer glibc
https://github.com/conda-forge/pytorch-cpu-feedstock/blob/master/recipe/meta.yaml#L35

Reply:
This is in the raw logs for pytoch builds
![image](https://user-images.githubusercontent.com/90008/149831059-fb263eab-b146-45ad-ae96-cf69544e25ce.png)


While this is in your builds
![image](https://user-images.githubusercontent.com/90008/149831077-26b03912-3e25-4fdd-b1eb-a427bea7cdb8.png)